cmd/kubelet/app/options/options.go

```diff
@@ -418,6 +422,8 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.BoolVar(&f.EnableCAdvisorJSONEndpoints, "enable-cadvisor-json-endpoints", f.EnableCAdvisorJSONEndpoints, "Enable cAdvisor json /spec and /stats/* endpoints.")
 	fs.MarkDeprecated("enable-cadvisor-json-apis", "will be removed in a future version")
 
+	fs.DurationVar(&f.MaxSurvivalGCAge.Duration, "maximum-container-ttl-duration", f.MaxSurvivalGCAge.Duration, "Maximum age for a finished container before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'")
```

Comment:
I feel the parameter name and description is not clear enough,  suppose we have a container that has ran for 10 days, then exited for 10 minutes,  this parameter should mean 10 minutes, not count the 10 days.

Reply:
+1 - using maybe `timeFromExit` or something?